src/d3a/setup/d3a-settings.json

```diff
@@ -21,6 +21,7 @@
       "EXPORT_ENERGY_TRADE_PROFILE_HR": false,
       "EXPORT_OFFER_BID_TRADE_HR": false,
       "RUN_REAL_TIME": false,
+      "KEEP_PAST_MARKETS": false,
```

Comment:
I think we can now safely deprecate this setting (i though i did already)

Reply:
:-) https://github.com/gridsingularity/d3a-interface/pull/118
Only one LGTM needed.

Reply:
I would rather be in favour of completely deprecating it once simulation results has completely be moved to a separate service